src/main/webapp/resources/js/pages/search/SearchSamplesTable.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useMemo } from "react";
+import { formatInternationalizedDateTime } from "../../utilities/date-utilities";
+import { getPaginationOptions } from "../../utilities/antdesign-table-utilities";
+import { Button, Table, TableProps } from "antd";
+import { SearchProject, SearchSample } from "./SearchLayout";
+import ProjectTag from "./ProjectTag";
+import { ColumnsType } from "antd/lib/table";
+import { SampleDetailViewer } from "../../components/samples/SampleDetailViewer";
+
+type SearchSamplesTableParams = {
+  samples:
+    | {
+        content: SearchSample[];
+        total: number;
+      }
+    | undefined;
+  handleTableChange: TableProps<SearchSample>["onChange"];
+};
+
+/**
+ * React component to render a table to display samples found in global search
+ * @param samples
+ * @param handleTableChange
+ * @constructor
+ */
+export default function SearchSamplesTable({
+  samples,
+  handleTableChange,
+}: SearchSamplesTableParams) {
+  const columns = useMemo<ColumnsType<SearchSample>>(
+    () => [
+      {
+        key: `identifier`,
+        dataIndex: "id",
+        title: i18n("SearchTable.identifier"),
+        width: 100,
+      },
+      {
+        key: `sampleName`,
+        dataIndex: "name",
+        title: i18n("SearchTable.name"),
+        sorter: true,
+        render: (name, sample) => {
+          return (
+            <SampleDetailViewer
```

Comment:
In tables I've been setting the button to launch the sample detail viewer to be a `link` with it's padding set to `0` rather than an actual button. So to keep it consistent can you change the button to a link (`<Button type="link">{name}</Button>`)

Reply:
Ok, updated in 68379c8